### PR TITLE
got: submission (new Portfile) Game of Trees 0.69

### PR DESCRIPTION
**Pull request by artkiver**
Closes: https://trac.macports.org/ticket/65059

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.3.1 21E258 arm64
Xcode 13.3.1 13E500a

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/got/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                got
+version             0.69
```
Comment (herbygillot) [inline devel/got/Portfile:6]: Let's add a `revision` field:

```suggestion
version             0.69
revision            0
```

**Comment by herbygillot** [comment]
Thanks for this submission @artkiver - I was going to comment that you could replace the `path:lib/libssl.dylib:openssl` dependency with the OpenSSL portgroup, then realized that perhaps one may want the option to use `libressl`.  Besides the comment above, this looks good.

**Comment by artkiver** [comment]
My apologies for the delayed reply, I haven't been near a keyboard much in the past week due to a lot of "life" happening. The revision field change looks good and hopefully I clicked the correct button to commit that change. As you observed, the dylib stanza is preferable as it facilitates the use of libressl for example (my initial Portfile [not submitted as a PR] actually specified LibreSSL explicitly, but I opted for the dynamic library approach to make it easier for those who may still be attached to OpenSSL or other TLS libraries).

**Comment by herbygillot** [comment]
Thanks @artkiver. This created a second commit and we usually like all changes to be a single commit.  I've resolved this by squashing the commits.

**Pull request merged**